recipes/catch2/2.x.x/conanfile.py

```diff
@@ -30,10 +38,16 @@ def config_options(self):
         if self.settings.os == "Windows" or not self.options.with_main:
             del self.options.fPIC
 
-    def configure(self):
+    def validate(self):
+        if tools.Version(self.version) < "2.13.1" and self.settings.arch == "armv8":
+            raise ConanInvalidConfiguration("ARMv8 is supported by 2.13.1+ only! give up!")
         if self.options.with_main and tools.Version(self.version) < "2.13.4":
             raise ConanInvalidConfiguration("Option with_main not supported with versions < 2.13.4")
 
+    def config_options(self):
+        if self.options.with_benchmark and not self.options.with_main:
+            del self.options.with_benchmark
+
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
```

Comment:
I know it's outside but since we have a PR open can you please can this function to this please
https://github.com/conan-io/conan-center-index/blob/4d22f7f6324210b449a413952e0a686f063c17b0/recipes/yojimbo/all/conanfile.py#L36